docroot/themes/custom/twentyeighteen/sass/base/_base.scss

```diff
@@ -35,80 +37,74 @@ body {
     }
   }
 }
+
+hr {
+  color: $teal;
+  border-width: 2pt;
+  border-style: solid;
+  margin: 2em 0;
+}
+
 // Base Headings
 //
 // Markup: headlines.twig
 //
 // Style guide: base.headlines
-h1 {
-  color: $sienna;
-  font-weight: 700;
-  font-family: $heading_font;
-}
-
+h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
-  color: $heading;
+  font-family: 'Montserrat', Helvetica, Arial, sans-serif;
```

Comment:
We should leave this as `$heading_font` and change the value of that variable in _config.scss